src/icon.c

```diff
@@ -39,6 +38,8 @@ static struct list_head icon_cache;
 
 static struct sbuf icon_theme;
 
+extern struct sbuf *cache_location;
```

Comment:
As discussed above, I prefer to remove this `extern` in favor of a `cache_get_dir()`. It's just tidier and seems less like using global variables.